code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -206,11 +209,17 @@
 /obj/item/weapon/card/id/New()
 	..()
 
+	id_cards += src
```

Comment:
what is this list currently used for?

Reply:
Its used in the account override ability